model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/nh_solve/solve_nonhydro_program.py

```diff
@@ -140,6 +143,9 @@ def predictor_stencils_2_3(
     vertical_start: gtx.int32,
     vertical_end: gtx.int32,
 ):
+    """
+    Temporary docstring to see if this program is picked up
+    """
```

Comment:
is this still needed?

Reply:
no because the patch doesn't work and this doesn't get picked up, I'll remove it